source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/actions/joint_actions_to_limits.py

```diff
@@ -185,7 +185,7 @@ def reset(self, env_ids: Sequence[int] | None = None) -> None:
             env_ids = slice(None)
         super().reset(env_ids)
         # reset history to current joint positions
-        self._prev_applied_actions[env_ids, :] = self._asset.data.joint_pos[env_ids, self._joint_ids]
+        self._prev_applied_actions[env_ids, :] = self._asset.data.joint_pos[env_ids][:, self._joint_ids]
```

Comment:
This should be faster since it does the indexing call once.

```suggestion
        self._prev_applied_actions[env_ids, :] = self._asset.data.joint_pos[env_ids[:, None], self._joint_ids]
```